docs/configuration.md

```diff
@@ -261,6 +261,7 @@ placement_policy:
 |------------------|----------|---------------|---------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `default`        | `string` | yes           | `REP 3`       | Default policy of placing containers in NeoFS. If a user sends a request `CreateBucket` and doesn't define policy for placing of a container in NeoFS, the S3 Gateway will put the container with default policy. |
 | `region_mapping` | `string` | yes           |               | Path to file that maps aws `LocationContraint` values to NeoFS placement policy. The similar to `--container-policy` flag in `neofs-s3-authmate` util, see in [docs](./authmate.md#containers-policy)             |
+| `regions`        | `map`    | yes           |               | The hashtable than maps aws `LocationContraint` values to NeoFS placement policy. The similar to `region_mapping` config option. The values extends/overrides values from region_mapping file, if it provided.    |
```

Comment:
Hashtable that maps aws `LocationConstraint` values to NeoFS placement policy. It's similar to the `region_mapping` config option, but allows for in-place configuration and extends/overrides values from `region_mapping` file if it's provided.

Reply:
Updated